adapters/user.transformer.ts

```diff
@@ -8,13 +8,19 @@ export const transformUsers = (users: any): User[] => {
 }
 
 export const transformUser = (user: any): User => {
+  const displayName = `${user.first_name} ${user.last_name}`;
+
   return {
     id: user.id,
+    displayName,
     username: user.username,
     firstName: user.first_name,
     lastName: user.last_name,
-    githubDisplayName: user.github_display_name,
-    githubId: user.github_id,
-    avatarUrl: user.picture?.url
+    avatar: user.picture?.url,
+    initials: displayName.trim()
+      .toUpperCase()
+      .split(" ", 2)
+      .map((str) => str.charAt(0))
+      .join(""),
```

Comment:
if we wanted initials only why took this long approach 
```
initials: 
`${user.first_name.chatAt(0) user.last_name.charAt(0)}`.toUpperCase()
```

Reply:
Make sense. Thanks for the suggestion.